styles/styles.css

```diff
@@ -41,6 +41,9 @@
   --icon-color: #205fff;
   --background-light-blue: #d9eeff;
   --background-blue: #006eff;
+  --tab-header-color-blue: #006eff;
```

Comment:
if these are used only on the features block you can add them to the features CSS file.